Sample-01/api-server.js

```diff
@@ -14,7 +14,7 @@ const appOrigin = authConfig.appOrigin || `http://localhost:${appPort}`;
 if (
   !authConfig.domain ||
   !authConfig.audience ||
-  authConfig.audience === "YOUR_API_IDENTIFIER"
+  ["{yourApiIdentifier}", "{API_IDENTIFIER}"].includes(authConfig.audience)
```

Comment:
Can "{API_IDENTIFIER}" be passed as a default value? I am just wondering where this is coming from as "{yourApiIdentifier}" seems to be the default value.

Reply:
I just looked at some python sample apps and saw that they have "{API_IDENTIFIER}" as the default value for API_IDENTIFIER. Please disregard the above comment.